ui/main.glade

```diff
@@ -466,6 +466,32 @@
                             <accelerator key="Delete" signal="activate"/>
                           </object>
                         </child>
+                        <child>
+                          <object class="GtkSeparatorMenuItem" id="separatormenuitem19">
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                          </object>
+                        </child>
+                        <child>
+                          <object class="GtkMenuItem" id="menuJournalMoveSelectionLayerUp">
```

Comment:
Nitpick: The `id` should have been renamed, since these menu items are under menu `Edit` now.
Also the PR description should have been updated accordingly (which is easy to do now).